docs/dev-notes/QA.md

```diff
@@ -4,8 +4,8 @@
 
 1. Make sure that the mobile app that you are testing with has SDK logging enabled. This is done with the following code:
 ```swift
-CustomerIO.initialize(siteId: Env.customerIOSiteId, apiKey: Env.customerIOApiKey, region: region) { config in
-    config.logLevel = .debug
+CustomerIO.initialize(writeKey: Env.cdpWriteKey) { config in
```

Comment:
Isn't it `Env.writeKey` instead? 🤔

Reply:
Now that I have changed `cdpWriteKey` to `writeKey` based on your suggestions on the other PR, I will update this key here as well.

Reply:
[Updated](https://github.com/customerio/customerio-ios/pull/513/commits/8d2e5c6dd6e1d2c51b0ef3bbe5d2e6546ccd6990)